ls1intum/Artemis
# `Programming exercises`: Update dependencies in Java templates

### Checklist
#### General

### Motivation and Context
Update default dependencies and bundled gradle version.

### Steps for Testing
1. Create a new Java programming language (once maven once gradle)
2. Check that executing test cases and SCA still works correctly.


### Review Progress
#### Code Review
#### Manual Tests
